playbooks/common/openshift-master/restart.yml

```diff
@@ -7,7 +7,7 @@
     openshift_master_ha: "{{ groups.oo_masters_to_config | length > 1 }}"
   serial: 1
   handlers:
-  - include: roles/openshift_master/handlers/main.yml
+  - include: ../../../roles/openshift_master/handlers/main.yml
```

Comment:
I don't like that we are pulling handlers out of roles like this, but that is another issue to address.